.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Ubuntu:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: Prepare
+      run: sudo apt update && sudo apt install -y unzip wget
+    - name: Download and extract Partiels binary
+      run: |
+        TAG=$(curl -s https://api.github.com/repos/Ircam-Partiels/Partiels/releases | jq -r '.[0].tag_name')
```

Comment:
I assume the python code is compatible with a specific version of Partiels, so the tag should be fixed, but we could also have a CI that is triggered periodically and checks if the latest release is compatible.